operator/deploy/csv/apicurio-registry.clusterserviceversion.yaml

```diff
@@ -0,0 +1,104 @@
+        version: v1
+  description: |
+    ## Apicurio Registry 3
+
+    Apicurio Registry stores and retrieves API designs and event schemas,
+    and gives you control of their evolution.
+
+    **Features**
+    - TODO
+
+    ## Apicurio Registry 3 Operator
+
+    Provides a quick and easy way to deploy and manage Apicurio Registry 3 on Kubernetes.
+
+    **Features**
+    - Supports installation and configuration of Apicurio Registry 3
+    - Perform a rolling upgrade of Apicurio Registry 3
+
+    ## Prerequisites
+
+    This operator does not deploy storage for Apicurio Registry 3. Therefore, some storage options require that the chosen persistence service is already set up.
+
+    ## License
+
+    Apicurio Registry 3 and Apicurio Registry 3 Operator are licensed under the [Apache 2.0 license](https://github.com/Apicurio/apicurio-registry/blob/main/LICENSE)
+  displayName: Apicurio Registry 3
+  icon:
+    - base64data: PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPCEtLSBHZW5lcmF0b3I6IEFkb2JlIElsbHVzdHJhdG9yIDI3LjEuMSwgU1ZHIEV4cG9ydCBQbHVnLUluIC4gU1ZHIFZlcnNpb246IDYuMDAgQnVpbGQgMCkgIC0tPgo8c3ZnIHZlcnNpb249IjEuMSIgaWQ9IkxheWVyXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IgoJIHZpZXdCb3g9IjAgMCAxODAgMjAwIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCAxODAgMjAwOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+CjxzdHlsZSB0eXBlPSJ0ZXh0L2NzcyI+Cgkuc3Qwe2ZpbGw6Izc1OEVCRjt9Cgkuc3Qxe2VuYWJsZS1iYWNrZ3JvdW5kOm5ldyAgICA7fQo8L3N0eWxlPgo8cGF0aCBkPSJNODEuOCwxMTIuOGMtMC4xLDAtMC4yLDAtMC4yLDB2MGMtOS43LTAuNi0xOC4xLTEwLjktMTguMS0xOC40YzAuMi01LjgsMC41LTEyLjgsMC44LTEzLjljNS40LTMwLjIsMjguMS01NC40LDU3LjQtNjIKCWMtMTAuNS01LjQtMjIuNC04LjUtMzUtOC41QzQ0LjQsMTAsMTAsNDQuNCwxMCw4Ni43VjE1NWMwLDE0LjgsMTIsMjYuNywyNi43LDI2LjdzMjYuNy0xMiwyNi43LTI2Ljd2LTE0LjNjMC0xMC40LDgtMTguOCwxOC4yLTE5LjUKCXYwYzAuMSwwLDAuMiwwLDAuMiwwYzIuMywwLDQuMi0xLjksNC4yLTQuMlM4NC4xLDExMi44LDgxLjgsMTEyLjhMODEuOCwxMTIuOHoiLz4KPGc+Cgk8cGF0aCBjbGFzcz0ic3QwIiBkPSJNMTQ2LjcsNDdjNC4xLDAsOC4xLDAuMywxMiwwLjljLTQuOS04LjMtMTEuMi0xNS42LTE4LjctMjEuNUMxMTIuNSwyOSw4OS4yLDQ1LjcsNzcuMyw2OS4xCgkJYzQuMS0zLjUsOS40LTUuNywxNS4yLTUuN2MxLjcsMCwzLjMsMC4yLDQuOSwwLjVDMTExLDUzLjQsMTI4LjEsNDcsMTQ2LjcsNDdMMTQ2LjcsNDd6Ii8+Cgk8cGF0aCBjbGFzcz0ic3QwIiBkPSJNMTY1LjMsNjIuN2MtNi45LTEuOS0xNC4xLTIuOS0yMS41LTIuOWMtMTMuMiwwLTI1LjcsMy4yLTM2LjcsOC45YzUuMyw0LjMsOC43LDEwLjgsOC43LDE4LjF2Ny44CgkJYzAsNy41LTguNCwxNy43LTE4LjEsMTguNHYwYy0wLjEsMC0wLjIsMC0wLjIsMGMtMi4zLDAtNC4yLDEuOS00LjIsNC4yczEuOSw0LjIsNC4yLDQuMmMwLjEsMCwwLjIsMCwwLjIsMHYwCgkJYzkuOCwwLjcsMTcuNiw4LjUsMTguMSwxOC40VjE1NWMwLDE0LjgsMTIsMjYuNywyNi43LDI2LjdzMjYuNy0xMiwyNi43LTI2LjdWODYuN0MxNjkuMiw3OC4zLDE2Ny44LDcwLjIsMTY1LjMsNjIuN0wxNjUuMyw2Mi43eiIKCQkvPgo8L2c+Cjwvc3ZnPgo=
+      mediatype: image/svg+xml
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: true
```

Comment:
`false` for now and double check later

Reply:
I've checked, we support `AllNamespaces` and `OwnNamespace`, and I'm 99% sure we support the other two as well, but the OCP UI does not.